README.rst

```diff
@@ -86,6 +86,7 @@ Tidelift will coordinate the fix and disclosure with maintainers.
 Maintainers
 -----------
 
+- `@Saiprasad16 <https://github.com/Saiprasad16>`__ (Saiprasad Kale)
```

Comment:
This list is for maintainers, not contributors. :) Please don't add to it.

Reply:
Ok :)